src/tools/items.ts

```diff
@@ -72,6 +72,7 @@ export const createGenericReadItemsTool = () => {
 							type: "text",
 							text: JSON.stringify({
 								error: `Error: ${error?.message || "Unknown error"}`,
+								wholeError: error,
```

Comment:
I haven't been able to find more info on this. Is `wholeError` the MCP spec for returning error objects?